### PR TITLE
Autodrobe Vox Outfits

### PR DESCRIPTION
**Pull request by falcon2346**
## What this does
Adds Vox Sprites for a handful of Autodrobe outfits, specifically the first 5 boxes that did not already have Vox sprites for all items. This PR includes sprites for the following:
Marisa outfit box: All Clothing Items
Captain's casual box: Green Glasses, Flatcap
Monkey suit box: Monkey Suit
Wizard outfit box: All Clothing Items
School girl outfit box: Kitty Ears, black skirt
dreamseeker_liIjENqaNg
Though pictured in this, the Jackboots, Green Suit, and Monkey Mask were not made by me, and are currently existing sprites in the game.

This also includes code contributed by @CrazyAmphibian to allow hats to hide the tail when given the HIDETAIL flag. Currently the only hat that uses this is the Kitty Ears.

Finally, somewhat unatomically, this fixes the Vox sprite for the Elite Security Skirt, which I happened to notice covered the hands.

## Why it's good
This gives more costume options for Vox clowns, as almost none fit properly currently. Also silly outfits are fun.

## How it was tested
I took the screenshots for the sprites in game.

## Changelog
[sprites]
:cl:
 * imageadd: Added Vox sprites for several Autodrobe items

### CODE REVIEW

**Comment by angryturnip** [comment]
Vox are pox

**Comment by Eneocho** [comment]
Vox are pox but monkeyvox is really funny to me

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*
+can /*
```
Comment (west3436) [inline code/modules/clothing/head/misc_special.dm]: Oops!

**Review by west3436 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/misc_special.dm (hunk 1 above)
Comment (falcon2346) [reply code/modules/clothing/head/misc_special.dm]: Done

**Review by west3436 — APPROVED** [review]

**Pull request merged**